### PR TITLE
python v3.12.6

### PR DESCRIPTION
**Pull request by h-vetinari**

### CODE REVIEW

**Comment by xhochy** [comment]
There could be some issues with the `if_runnable.sh` script:

```
/home/conda/feedstock_root/build_artifacts/python-split_1725950768409/work/build-static/_PYTHON_PROJECT_BASE=/home/conda/feedstock_root/build_artifacts/python-split_1725950768409/work/build-static: No such file or directory
```

File: recipe/patches/0020-Don-t-checksharedmods-if-cross-compiling.patch [hunk 1]
```diff
@@ -24,17 +24,12 @@ index 689ece9d24..913cbd8e23 100644
...
 +#!/bin/bash
 +
 +if [[ "${target_platform}" == "${build_platform}" || "${CROSSCOMPILING_EMULATOR:-}" != "" ]]; then
-+  if [[ "$1" == "./"* ]]; then
-+    exec $@
-+  else
-+    export ${@:1:$#-2}
-+    exec ${@: -2}
-+  fi
++    exec $*
```
Comment (xhochy) [inline recipe/patches/0020-Don-t-checksharedmods-if-cross-compiling.patch:34]: @h-vetinari This diff looks at fault.

Comment (h-vetinari) [reply recipe/patches/0020-Don-t-checksharedmods-if-cross-compiling.patch:34]: Strange, I didn't touch that patch when rebasing, that was git's doing.

Comment (h-vetinari) [reply recipe/patches/0020-Don-t-checksharedmods-if-cross-compiling.patch]: Ah, nevermind, the reason was that I hadn't picked the fixes from #696 into my local branch 🤦

**Review by xhochy — APPROVED** [review]

**Pull request merged**